README.rst

```diff
@@ -132,6 +132,20 @@ tutorial <http://github.com/blaze/blaze-tutorial>`__, `some
 blogposts <http://continuum.io/blog/tags/blaze>`__, or the `mailing list
 archives <https://groups.google.com/a/continuum.io/forum/#!forum/blaze-dev>`__.
 
+
+Development setup
+=================
+
+The quickest way to install all Blaze dependencies with `conda` is as follows::
+
+    conda install blaze spark dynd-python -c libdynd -c blaze -c anaconda-cluster -y
+    conda remove odo blaze blaze-core datashape -y
+
+After running these commands, clone `odo`, `blaze`, and `datashape` from GitHub
+directly.  These three projects release together.  Run `python setup.py
```

Comment:
Can you put double backticks here. That'll make it show up as a fixed width font.